apps/playground-lit-e2e/src/e2e/form/form.cy.ts

```diff
@@ -0,0 +1,37 @@
+const playgroundLitUrl = 'http://localhost:4202';
+
+describe('integration Form - Lit', () => {
+    it('should be accessible', () => {
+        cy.visitWithA11y(playgroundLitUrl);
+
+        cy.checkA11y('app-element');
+    });
+
+    it('should be CSP compliant', () => {
+        cy.intercept('/csp-report', cy.spy().as('cspReport'));
+        cy.visit(playgroundLitUrl);
+
+        // eslint-disable-next-line cypress/no-unnecessary-waiting
+        cy.wait(500);
+        cy.get('@cspReport').should('not.have.been.called');
+    });
+
+    it('should report CSP violation', () => {
```

Comment:
de test die CSP violation zou moeten faalt lokaal

<img width="1063" alt="image" src="https://github.com/milieuinfo/uigov-web-components/assets/15731348/1690adb2-4eda-4d29-88fc-9621511c89d6">

maar slaagt wel op bamboo

<img width="1292" alt="image" src="https://github.com/milieuinfo/uigov-web-components/assets/15731348/76e46f38-1703-47b7-bd6d-573385ee9edd">

weet niet in welke mate dit gewoon aan mijn systeem ligt

---

update: bij Tony werkt het wel perfect lokaal dus ligt aan mijn systeem

![image](https://github.com/milieuinfo/uigov-web-components/assets/15731348/8b70aaa1-cdfc-4641-9582-8074bfcc16d2)

Reply:
Hmm wel best vreemd. Met welke browser heb je de testen opgestart? Welke browser versie hebben jullie beide?

Reply:
zelfde browser in headless iig

Reply:
Dan moeten we eens zien wat er bij jou misloopt.
Ik ga de branch alvast mergen aangezien het probleem enkel bij jou optreedt.